shared/chat/conversation/giphy/index.native.js

```diff
@@ -1,8 +1,45 @@
 // @flow
+import * as React from 'react'
+import * as Kb from '../../../common-adapters'
+import {NativeWebView} from '../../../common-adapters/native-wrappers.native'
+import * as Styles from '../../../styles'
 import type {Props} from './index.types'
 
-const GiphySearch = (props: Props) => {
-  return null
+class GiphySearch extends React.Component<Props, State> {
+  render() {
+    const vidStyle = 'height: 100%25 ;'
```

Comment:
i think this should be hardcoded on the results page instead of passing through as a URL param. 1. in makes the code cleaner since you don't have to parse it back out and 2. i feels weird/wrong to be echoing client params into HTML, can you set the `Styles.globalColors.black_20` in react move everything else to goland?

Reply:
If it is in Go, it makes changing it very time consuming, since you need to recompile everytime you want to tweak something. This way I can just reload the JS and get results right away.